src/+otp/+sanzserna/+presets/Canonical.m

```diff
@@ -0,0 +1,11 @@
+classdef Canonical < otp.sanzserna.SanzSernaProblem
+    methods
+        function obj = Canonical(m)
+            if nargin < 1
+                m = 32;
+            end
+            x = linspace(1/m + 1, 2, m).';
```

Comment:
Do we need this to be transposed here?

Reply:
In fact, this should be called "y0". (And it does need to be transposed, yeah.)

Reply:
Are you sure that you are not mixing this up with the "x" used in the problem? What is defined here should be the initial condition, and be named as such.

Reply:
Yes let's rename to `y0`.